app/src/ui/changes/changed-file.tsx

```diff
@@ -58,8 +58,14 @@ export class ChangedFile extends React.Component<IChangedFileProps, {}> {
       filePadding -
       statusWidth
 
+    const screenReaderDescription = `${path}  ${mapStatus(status)}`
+
     return (
-      <div className="file" onContextMenu={this.onContextMenu}>
+      <div
+        className="file"
+        onContextMenu={this.onContextMenu}
+        aria-label={screenReaderDescription}
```

Comment:
@tidy-dev looks like I missed that this is also trying to use `aria-label` on a generic HTML element (a `div` or a `span`) without any semantics attached to it (it's not marked up as `role="button"`, `role="link"`, etc.) That's likely also contributing to the mysterious behavior -- `aria-label` on a non-interactive element just doesn't consistently work. You can see [our guidance on a corresponding ERBLint rule](https://github.com/github/erblint-github/blob/main/docs/rules/accessibility/no-aria-label-misuse-counter.md) for some more info/suggested remediation (unfortunately I don't think the ESLint plugin we borrowed from elsewhere has this rule yet).

Based on this code, would you describe this element as a button (it seems there's a  click handler on it)? If so, I'd try adding `role="button"; `tabindex=0`; and a focus indicator. That'll help with keyboard accessibility and the explicit `role=button` would likely help with getting this label read out.